lib/generators/active_admin/install/templates/migrations/create_active_admin_comments_rails5.rb

```diff
@@ -0,0 +1,19 @@
+class CreateActiveAdminComments < ActiveRecord::Migration[5.0]
```

Comment:
This would be better:

```ruby
parent_class = ActiveRecord::Migration
parent_class = parent_class[5.0] if Rails::VERSION::MAJOR >= 5

class CreateActiveAdminComments < parent_class